scripts/Diffraction/isis_powder/routines/common.py

```diff
@@ -252,6 +267,16 @@ def generate_unsplined_name(vanadium_string, *args):
     return _generate_vanadium_name(vanadium_string, False, *args)
 
 
+def generate_summed_empty_name(empty_runs_string):
```

Comment:
does there need to be a a private form of this function?

Reply:
Probably not - I was mimicking the approach in the functions generate_unsplined_name and generate_splined_name which are right above this in the same unit and they both call a private function. Looking at it again though these two both call the same private function so there's some logic\benefit to the private function. I'll get rid of the private function here and just put the code in generate_summed_empty_name